docs/estimate.md

```diff
@@ -202,7 +202,7 @@ Tezos.estimate
 
 ```js live noInline wallet
 // import { TezosToolkit } from '@taquito/taquito';
-// const Tezos = new TezosToolkit('https://jakartanet.api.tez.ie');
+// const Tezos = new TezosToolkit('https://kathmandunet.api.tez.ie');
```

Comment:
`https://kathmandunet.api.tez.ie` is not a valid url

https://kathmandunet.ecadinfra.com

Reply:
Fixed. also fixed another case in making_transfer.md